common/api/core-backend.api.md

```diff
@@ -5944,6 +5976,16 @@ export class TxnManager {
     protected _onGeometryGuidsChanged(changes: ModelIdAndGeometryGuid[]): void;
     readonly onModelGeometryChanged: BeEvent<(changes: ReadonlyArray<ModelIdAndGeometryGuid>) => void>;
     readonly onModelsChanged: BeEvent<(changes: TxnChangedEntities) => void>;
+    // @internal (undocumented)
+    protected _onRebaseLocalTxnConflict(internalArg: DbRebaseChangesetConflictArgs): DbConflictResolution;
+    // @internal (undocumented)
+    readonly onRebaseTxnBegin: BeEvent<(txn: TxnArgs) => void>;
+    // @internal (undocumented)
+    protected _onRebaseTxnBegin(txn: TxnArgs): void;
+    // @internal (undocumented)
+    readonly onRebaseTxnEnd: BeEvent<(txn: TxnArgs) => void>;
+    // @internal (undocumented)
+    protected _onRebaseTxnEnd(txn: TxnArgs): void;
```

Comment:
TxnManager isn't intended to be subclassed. If you make these private instead of protected then @internal is not needed.

Reply:
I made the callbacks that I added to private.